src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const truePosition = {};
+
+  for (const key in robot) {
```

Comment:
It's good practice to check if the property belongs to the object itself and not to its prototype chain. Use 'robot.hasOwnProperty(key)' to ensure that the key is a direct property of 'robot'.